pkgs/applications/networking/netmaker/default.nix

```diff
@@ -4,7 +4,7 @@
 , lib
 , libglvnd
 , pkg-config
-, subPackages ? ["." "netclient"]
+, subPackages ? [ "." ]
```

Comment:
with this change, `netmaker-full` will end up being a duplicate of `netmaker`, is that what we want here?

Reply:
I think we want to remove the `netmaker-full` package. Or rework it in a way where it will also build the new `netclient`.

Reply:
Honestly I'm new to Nixos and it's ecosystem, so I'm not sure what's the correct call here.